### PR TITLE
REPL blog post

### PR DESCRIPTION
**Pull request by drwpow**
## Changes
Announces the REPL!

## Testing

👀 https://astro-www-git-blog-post-astro-repl-pikapkg.vercel.app/blog/astro-repl

## Docs

It's a blog post

### CODE REVIEW

File: www/src/pages/blog/astro-repl.astro [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <head>
+    <BaseHead title={title} description={description} permalink={permalink} />
+    <link rel="stylesheet" href={Astro.resolve('../../scss/blog.css')} />
+  </head>
+
+  <body>
+    <BlogHeader />
+    <BlogPost title={title} author="drew" heroImage="/assets/blog/astro-repl/astro-repl-hero.jpg" publishDate={publishDate}>
+      <Markdown>
+        The Astro team proudly presents the new Astro REPL: write Astro code straight
```
Comment (FredKSchott) [inline www/src/pages/blog/astro-repl.astro:24]: Here's my suggested edits, some for style and some to fix some grammar issues. Also, added a link to the Demo Day live stream as a secondary call to action.

Please rewrite/re-edit in your own voice!

        The Astro team proudly presents the new Astro REPL: write Astro code right in your browser. 
        Use this new REPL to try out Astro's HTML component syntax, prototype your next big idea, 
        or debug issues with ease. Under the hood, the REPL is powered by Astro’s brand new 
        WASM compiler (written in Go) that can rebuild your code faster than you can write it.

        Play around with the Astro REPL today at [astro.build/play](https://astro.build/play). 
        To learn more about our new Go-powered compiler, [join us on Discord](https://astro.build/chat) and tune in for [Astro Demo Days](https://www.youtube.com/watch?v=-ExcBJrXOd8) next Monday at 11am PST.

**Review by FredKSchott — APPROVED** [review]
LGTM! One more thing: can you update the homepage link to point to this post? The 0.19.0 release post is getting a bit stale.

File: www/src/pages/blog/astro-repl.astro (hunk 1 above)
Comment (natemoo-re) [reply www/src/pages/blog/astro-repl.astro:24]: I would use "Monday, September 20, 2021" rather than "next Monday", but otherwise like your suggestions!

Comment (FredKSchott) [reply www/src/pages/blog/astro-repl.astro:24]: yes good call!

**Comment by FredKSchott** [comment]
@drwpow sounds like you've stepped away for a bit, i'll assume comments are fine and will merge and get out.

**Pull request merged**